boilerplate/app/screens/DemoShowroomScreen/DemoShowroomScreen.tsx

```diff
@@ -181,22 +183,30 @@ export const DemoShowroomScreen: FC<DemoTabScreenProps<"DemoShowroom">> =
           </View>
         )}
       >
-        <Screen preset="fixed" safeAreaEdges={["top"]} contentContainerStyle={$styles.flex1}>
+        <Screen
+          preset="fixed"
+          safeAreaEdges={["top"]}
+          contentContainerStyle={$styles.flex1}
+          {...(isAndroid ? { KeyboardAvoidingViewProps: { behavior: undefined } } : {})}
+        >
           <DrawerIconButton onPress={toggleDrawer} />
 
-          <SectionList
+          <SectionListWithKeyboardAwareScrollView<ReactElement[]>
```

Comment:
I just noticed this, why does this tag appear to have a type annotation? I've never seen this with JSX before

Reply:
Addressed in https://github.com/infinitered/ignite/pull/2722/commits/b1c1525bc37afc14950aed71d13c8ca8dc8d6419.

After investigation, I found that it's typical for devs consuming `SectionList` to allow `ItemT` to remain as `any` and narrow types in `renderItem`. To support that, I updated the component to default to `any` for that type.